src/hard-tabs-transform.coffee

```diff
@@ -1,8 +1,10 @@
+require("es6-shim")
```

Comment:
Since this is required for a side-effect, I wonder if we should require it in the main module for this library. That doesn't currently exist, but we could require it in `spec-helper`.